static/bidder-info/smarthub.yaml

```diff
@@ -0,0 +1,13 @@
+maintainer:
+  email: "support@smart-hub.io"
```

Comment:
Just double-checking: are you registered with IAB Europe? If you don't, that's fine. If you do, IAB's GVL id number must be written into this file as shown below. Just double-checking in case you are.
```
 1   maintainer:
 2     email: "support@smart-hub.io"
   + gvlVendorID: <IABs_GVL_ID>
 3   capabilities:
 4     app:
 5       mediaTypes:
 6         - banner
 7         - video
 8         - native
 9     site:
10       mediaTypes:
11         - banner
12         - video
13         - native
static/bidder-info/smarthub.yaml
```

Reply:
We don't registered with IAB Europe

Reply:
SmartHub is the solution for different companies which every has own registration and Vendor ID. Every company which receive traffic will process European data by itself. If a company doesn't have Vendor ID, SmartHub clean all data due to IAB recommendations. SmartHub completely support GDPR, but I think it's incorrect to write our Vendor ID here